src/main/java/gregtech/common/blocks/BlockTurbineCasing.java

```diff
@@ -29,30 +31,42 @@ public boolean canCreatureSpawn(IBlockState state, IBlockAccess world, BlockPos
         return false;
     }
 
-    public enum TurbineCasingType implements IStringSerializable {
+    public enum TurbineCasingType implements IStringSerializable, IStateHarvestLevel {
```

Comment:
If this implements `IStateHarvestLevel`, it should probably not call `setHarvestLevel` in the constructor.